src/BehaviorsSDKManaged/Microsoft.Xaml.Interactions/Core/TransitionControlAction.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Microsoft.Xaml.Interactivity;
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Animation;
+
+namespace Microsoft.Xaml.Interactions.Core
+{
+    public enum AnimationKind
+    {
+        Right,
+        Left,
+        Up,
+        Down
+    }
+    public sealed class TransitionControlAction : DependencyObject, IAction
+    {
+
+
+        public AnimationKind AnimationKind
+        {
+            get { return (AnimationKind)GetValue(AnimationKindProperty); }
+            set { SetValue(AnimationKindProperty, value); }
+        }
+
+        // Using a DependencyProperty as the backing store for AnimationKind.  This enables animation, styling, binding, etc...
+        public static readonly DependencyProperty AnimationKindProperty =
+            DependencyProperty.Register("AnimationKind", typeof(AnimationKind), typeof(TransitionControlAction), new PropertyMetadata(AnimationKind.Right));
+
+
+        public TimeSpan Duration
+        {
+            get { return (TimeSpan)GetValue(DurationProperty); }
+            set { SetValue(DurationProperty, value); }
+        }
+
+        // Using a DependencyProperty as the backing store for Duration.  This enables animation, styling, binding, etc...
+        public static readonly DependencyProperty DurationProperty =
+            DependencyProperty.Register("Duration", typeof(TimeSpan), typeof(TransitionControlAction), new PropertyMetadata(TimeSpan.FromMilliseconds(500)));
+
+
+        private void AnimateControl(FrameworkElement control, TimeSpan duration, AnimationKind kind)
```

Comment:
General usage of `var` is against https://github.com/Microsoft/XamlBehaviors/wiki/Contributing-Guidelines#c